api/src/main/java/org/eclipse/microprofile/config/inject/package-info.java

```diff
@@ -44,5 +44,5 @@
  * @author <a href="mailto:struberg@apache.org">Mark Struberg</a>
  * @author <a href="mailto:emijiang@uk.ibm.com">Emily Jiang</a>
  */
-@org.osgi.annotation.versioning.Version("1.1")
+@org.osgi.annotation.versioning.Version("2.0")
```

Comment:
How about if we align these versions?

Reply:
This is the semantic version for the corresponding package. They are all individually version to indicate whether there are any backward incompatible changes between teh releases. There should not be any alignment with the different packages. As you might have noticed, not all of the packages have changed their versions. For more [info](http://docs.osgi.org/whitepaper/semantic-versioning/040-semantic-versions.html) regarding semantic versioning. We use bnd versioning plugin to work out all of the versions.

Reply:
I know, but this is almost like starting fresh, and I think it wouldn't hurt to align all versions (even if there are no changes, but to mark a baseline with the Jakarta rename) and then follow the regular versioning scheme going forward.

Reply:
It's not like starting from scratch though. We have been follow the regular versioning scheme from the beginning. I think it is expecially important to obey the rules as you can see some packages are not even affected by the changes so their version will not be updated. If we directly jump from 1.1->3.0, this will confuse end users.

Reply:
@radcortez do you have further comments? If not, can you approve this PR so that I can make further progress. At the moment, I am blocked by this.

Reply:
Package versions should be semantic. Bundle versions and specification document versions are free to change. 

If you change the major version of a package then you affect all consumers of the package by breaking their imports as they, properly, import to exclude a major version bump as that means a breaking change.

Each package has an independent semantic versioning life. There is no reason to keep all package versions in lock step and there is no reason to change a major version unless there is a breaking change you need to communicate to users of the package.

A package's version is a message to the future. Make sure the message is clear and not noise.